lib/stack_master/aws_driver/cloud_formation.rb

```diff
@@ -7,13 +7,24 @@ def region
         @region ||= ENV['AWS_REGION'] || Aws.config[:region] || Aws.shared_config.region
       end
 
+      def profile_name
+        @profile_name ||= ENV['AWS_PROFILE'] || Aws.config[:profile_name]
+      end
```

Comment:
I'd like to see this in a seperate PR to have a seperate conversation there. I'm not convinced this is the right path for StackMaster.

Reply:
I could split this out. Do you not think the whole PR is not the right direction or just this change?

Specifying the profile is completely optional. If you leave it out it will just default to the env variables.

Reply:
I understand but I believe a Pull request should have [one concern](https://medium.com/@fagnerbrack/one-pull-request-one-concern-e84a27dfe9f1) and one concern only.

Discussions of this change are likely to confuse the other discussion that needs to happen.